src/org/usfirst/frc4904/robot/subsystems/BallIO.java

```diff
@@ -0,0 +1,25 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class BallIO extends Subsystem {
+	public SpeedController topMotor;
```

Comment:
Make these instances of `Motor` rather than `SpeedController`
There're a couple of benefits:
- default command is `MotorIdle`, so you don't have to stop the motors when you're done
- you can group speed controllers together
- you can add speed modifiers
and so on...